packages/shared/lib/core/router/app-router.ts

```diff
@@ -148,14 +154,18 @@ export class AppRouter extends Router<AppRoute> {
                 }
                 break
             case AppRoute.Import: {
-                const { importType } = params
+                const { importType, strongholdUpdateRequired } = params
```

Comment:
strongholdUpdateRequired is always false in my case, since params is missing the property:
<img width="992" alt="image" src="https://user-images.githubusercontent.com/6748220/224688353-0e130c07-f50a-4500-9e0a-61b300a54b94.png">